libsolidity/Compiler.cpp

```diff
@@ -282,6 +292,21 @@ void Compiler::registerStateVariables(ContractDefinition const& _contract)
 			m_context.addStateVariable(*variable);
 }
 
+void Compiler::generateAccessorCode(VariableDeclaration const& _varDecl)
+{
+	m_context.startNewFunction();
+	m_returnTag = m_context.newTag();
+	m_breakTags.clear();
+	m_continueTags.clear();
+
+	m_context << m_context.getFunctionEntryLabel(_varDecl);
+	ExpressionCompiler::appendStateVariableAccessor(m_context, &_varDecl);
+
+	uint64_t foo = uint64_t(_varDecl.getType()->getStorageSize());
```

Comment:
I guess I was playing around with that code too much. uint64_T was since `getStorageSize()` returned `u256` but if I use the function you suggested I get an `unsigned` which will solve everything :)